src/ui/src/app/dashboard/dashboard-body/main/main.component.ts

```diff
@@ -1,6 +1,8 @@
 import { Component } from '@angular/core';
 import { CalendarViewComponent } from "./attendance-module/calendar-view/calendar-view.component";
 import { UpdateLeaveSettingsComponent } from './leave-management/update-leave-settings/update-leave-settings.component';
+import { LeaveApplicationComponent } from "./leave-management/leave-application/leave-application.component";
```

Comment:
why?

Reply:
it was there in the html,but not imported.So to avoid error,i just import it.